.github/workflows/python.yml

```diff
@@ -31,29 +56,52 @@ jobs:
     - name: Install cargo-llvm-cov
       uses: taiki-e/install-action@cargo-llvm-cov
 
-    - name: Install dependencies
+    - name: Create and activate virtual environment (Unix)
+      if: runner.os != 'Windows'
       run: |
         python -m venv .venv
-        .venv/bin/python -m pip install --upgrade pip
-        .venv/bin/pip install -r requirements.txt
-        .venv/bin/pip install pytest-cov
+        echo "VIRTUAL_ENV=$(pwd)/.venv" >> $GITHUB_ENV
+        echo "$(pwd)/.venv/bin" >> $GITHUB_PATH
+
+    - name: Create and activate virtual environment (Windows)
+      if: runner.os == 'Windows'
+      run: |
+        python -m venv .venv
+        echo "VIRTUAL_ENV=$(pwd)/.venv" >> $GITHUB_ENV
+        echo "$(pwd)/.venv/Scripts" >> $GITHUB_PATH
+
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install -r requirements.txt
+        python -m pip install pytest-cov
+
     - name: Install Django Rusty Templates
       run: |
-        source <(cargo llvm-cov show-env --export-prefix)
-        .venv/bin/maturin develop
+        cargo llvm-cov show-env --export-prefix > llvm-cov-env.sh
+        source llvm-cov-env.sh
+        maturin develop
+
     - name: Build translation files
-      run: .venv/bin/python -m django compilemessages
+      run: |
+        python -m django compilemessages
+
     - name: Lint test files
       run: |
-        .venv/bin/ruff check
+        ruff check
+
     - name: Test with pytest
       run: |
-        source <(cargo llvm-cov show-env --export-prefix)
-        .venv/bin/pytest --cov --cov-report=xml
+        cargo llvm-cov show-env --export-prefix > llvm-cov-env.sh
+        source llvm-cov-env.sh
+        pytest --cov --cov-report=xml -v
```

Comment:
Ah, the `-v` verbose flag can be removed, this was to see a bit more output from the pytest failure